Lib/distutils/command/check.py

```diff
@@ -80,8 +80,11 @@ def run(self):
     def check_metadata(self):
         """Ensures that all required elements of meta-data are supplied.
 
-        name, version, URL, (author and author_email) or
-        (maintainer and maintainer_email)).
+        required fields:
```

Comment:
We can probably leave this as-is, since the substantive behavior hasn't changed, just the wording, but I'm also OK with keeping this split out as well. If we keep it split out, though, the "required" and "recommended" should be capitalized, and I think the fields should be indented and not end in a period, like so:

```
Required fields:
    name, version, URL

Recommended fields:
    (author and author_email) or (maintainer and maintainer_email)
```

Reply:
Thank you for the suggestion.